app/src/main/java/com/example/poste/models/Folder.java

```diff
@@ -52,6 +53,16 @@ public String getTitle() {
         return title;
     }
 
+    public Post getPostFromFolder(String postId){
```

Comment:
Nicely done. You might consider placing a method on User, perhaps two with one overloaded with FolderID; that's a matter of preference and abstraction level though.

Reply:
Partially done, can expand in future